docs/source/api/pystiche_papers.li_wand_2016.rst

```diff
@@ -27,31 +27,149 @@
+described in the paper. The following parts are affected:
+
+- :class:`~pystiche_papers.li_wand_2016.FeatureReconstructionOperator`
+- :class:`~pystiche_papers.li_wand_2016.MRFOperator`
+- :class:`~pystiche_papers.li_wand_2016.TotalVariationOperator`
+- :class:`~pystiche_papers.li_wand_2016.target_transforms`
+
+The behavior can be switched with the ``impl_params`` flag, which defaults to ``True``.
+
+Hyper parameters
+----------------
+
+Furthermore, the :func:`~pystiche_papers.li_wand_2016.hyper_parameters` also mismatch.
+The ``impl_params`` flag is used to switch between between these two sets.
+
+
+:func:`~pystiche_papers.li_wand_2016.content_loss`
+``````````````````````````````````````````````````
+
++------------------+-----------------+-----------+
+| Parameter        | ``impl_params``             |
++                  +-----------------+-----------+
+|                  | ``True``        | ``False`` |
++==================+=================+===========+
+| ``layer``        | ``"relu4_2"``               |
++------------------+-----------------+-----------+
+| ``score_weight`` | ``2e1``         | ``1e0``   |
++------------------+-----------------+-----------+
+
+
+:func:`~pystiche_papers.li_wand_2016.target_transforms`
+```````````````````````````````````````````````````````
+
++-----------------------+-----------------+-----------+
+| Parameter             | ``impl_params``             |
++                       +-----------------+-----------+
+|                       | ``True``        | ``False`` |
++=======================+=================+===========+
+| ``num_scale_steps``   | ``0``           | ``3``     |
++-----------------------+-----------------+-----------+
+| ``scale_step_width``  | ``5e-2``                    |
++-----------------------+-----------------+-----------+
+| ``num_rotate_steps``  | ``0``           | ``2``     |
++-----------------------+-----------------+-----------+
+| ``rotate_step_width`` | ``7.5``                     |
++-----------------------+-----------------+-----------+
+
+:func:`~pystiche_papers.li_wand_2016.style_loss`
+````````````````````````````````````````````````
+
++-------------------+-----------------+-----------+
+| Parameter         | ``impl_params``             |
++                   +-----------------+-----------+
+|                   | ``True``        | ``False`` |
++===================+=================+===========+
+| ``layers``        | ``("relu3_1", "relu4_1")``  |
++-------------------+-----------------+-----------+
+| ``layer_weights`` | ``"sum"``                   |
++-------------------+-----------------+-----------+
+| ``patch_size``    | ``3``                       |
++-------------------+-----------------+-----------+
+| ``stride``        | ``2``           | ``1``     |
++-------------------+-----------------+-----------+
+| ``score_weight``  | ``1e-4``        | ``1e0``   |
++-------------------+-----------------+-----------+
```

Comment:
`score_correction_factor` is missing. 

However, this is currently being discussed in [#228](https://github.com/pmeier/pystiche_papers/issues/228).